components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/workflow/ApplicationCreationWSWorkflowExecutor.java

```diff
@@ -169,58 +142,130 @@ public void complete(WorkflowDTO workFlowDTO) throws WorkflowException {
                 try {
                     dao.updateApplicationStatus(Integer.parseInt(workFlowDTO.getWorkflowReference()), status);
                 } catch (APIManagementException e) {
-                    String msg = "Error occured when updating the status of the Application creation " + "process";
+                    String msg = "Error occurred when updating the status of the Application creation " + "process";
                     log.error(msg, e);
                     throw new WorkflowException(msg, e);
                 }
-                
-            }else {
+
+            } else {
                 String msg = "Application does not exist";
                 throw new WorkflowException(msg);
             }
         } catch (APIManagementException e) {
-            String msg = "Error occured when retrieving the Application creation with workflow ID :"+workFlowDTO.getWorkflowReference();
+            String msg = "Error occurred when retrieving the Application creation with workflow ID :" + workFlowDTO
+					.getWorkflowReference();
             log.error(msg, e);
             throw new WorkflowException(msg, e);
         }
     }
 
-	@Override
-	public List<WorkflowDTO> getWorkflowDetails(String workflowStatus) throws WorkflowException {
-		// TODO Auto-generated method stub
-		return null;
-	}
+    @Override
+    public List<WorkflowDTO> getWorkflowDetails(String workflowStatus) throws WorkflowException {
+        // TODO Auto-generated method stub

```

Comment:
If we do not implement this method please add a meaningful comment and remove the default generated comment.